avocado/core/job.py

```diff
@@ -319,10 +319,14 @@ def _log_avocado_version():
         job_log = _TEST_LOGGER
         job_log.info('Avocado version: %s', version.VERSION)
         if os.path.exists('.git') and os.path.exists('avocado.spec'):
-            cmd = "git show --summary --pretty='%H' | head -1"
-            status, top_commit = commands.getstatusoutput(cmd)
+            cmd = "git show --summary --pretty='%H'"
+            result = process.run(cmd, ignore_status=True)
```

Comment:
@ldoktor if you consider this (not using `avocado.utils.process`) to be important to the project as a whole, I suggest you turn this preference you have into a RFC/PR.

In it, you'd some profiling, publish performance statistics, advocate the changes (`s/avocado.utils.process.*/subprocess.*/`) and write documentation targeted to our contributors and users (test writers) clearly stating that they should favor `subprocess` for "simple" use cases.

IMO, while you do that, you'd probably end up fixing any overhead that `avocado.utils.process` may have.  I myself don't feel compeled to do this.  But IMO, it's how your preference should be expressed here.

So the question is: can we expect to see a RFC/PR?